src/components/SearchFilter/SearchFilter.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from "react";
+import { useForm } from "react-hook-form";
+import { useHistory } from "react-router-dom";
+import { Select } from "../Select/Select";
+import { DataList } from "../DataList/DataList";
+import { SERVICES, DATALIST, PETS } from "../../constants/SearchOptions";
+import AnnouncementsContext from "../../context/announcements/announcements-context";
+import styles from "./SearchFilter.module.scss";
+
+export const SearchFilter = () => {
+  const history = useHistory();
+  const announcementsCtx = useContext(AnnouncementsContext);
```

Comment:
If you call it `annoucementsContext` there won't be any problems when the real context is called `AnnoucementsContext`. It is case-sensitive.